### ISSUE TITLE
add support for `deprecated` parameter

### ISSUE THREAD

**Issue by maciekpitucha** [PaloAltoNetworks/docusaurus-openapi-docs#360]
### Is your feature request related to a problem?

[OpenApi 3.0.3](https://swagger.io/specification/#schema-object) allows for a parameter `deprecated` in the schema object but the extension does not seem to use it.

### Describe the solution you'd like

Property name could be crossed out and word "deprecated" can be used somewhere next to it, preferably in a colour that stands out.

### Describe alternatives you've considered

Technically, it could be handled by the description:
```json
"description": "<span style='color: orange'>deprecated</span> User's email",
```

but since the schema actually allows for `deprecated`, using it seems cleaner.

**Comment by sserrata** [comment]
Hi @maciekpitucha, thanks for the feature request. I think we should be able to adapt a solution based on how we mark properties as "required".

**Comment by sserrata** [comment]
Hi @maciekpitucha, I opened a PR to introduce support for deprecating schema items/properties. Let me know if that's what you had in mind. Thanks!

**Comment by sserrata** [comment]
Available to test with `0.0.0-530` release. Let me know what you think. Thanks!
